### DISCUSSION TITLE
Speaker-diarization

### DISCUSSION THREAD

**Discussion by burtrron** [Dadangdut33/Speech-Translate#19] [Ideas]
Dadangdut33,

Thank you so much for this great tool. I use it daily to save me lots of time with transcribing audio and video recordings. I am so grateful for your efforts. 

An additional feature that would be incredibly helpful for me, and perhaps others, would be speaker-diarization. My recordings typically include two to four participants, and I currently manually differentiate between them. Would speaker-diarization be something you could integrate? I don’t have the aptitude to code it myself. 

Thanks so much in advance. I appreciate your work.

**Comment by Dadangdut33** [comment]
Great idea! i have seen other people done this so it might be possible and it is something that i eventually wanted to try adding. 

Unfortunately i'm quite busy right now but I'll be sure to try adding this and i'll update to let you know if it will be possible or not

**Comment by burtrron** [comment]
Thank you so much!
